pkgs/by-name/ri/rime-wanxiang/package.nix

```diff
@@ -1,60 +1,35 @@
 {
   lib,
-  fetchFromGitHub,
   stdenvNoCC,
-  librime,
-  rime-data,
-  nix-update-script,
+  fetchFromGitHub,
+  gitUpdater,
 }:
 
 stdenvNoCC.mkDerivation (finalAttrs: {
   pname = "rime-wanxiang";
-  version = "11.3.1";
+  version = "11.3.2";
 
   src = fetchFromGitHub {
     owner = "amzxyz";
     repo = "rime_wanxiang";
     tag = "v" + finalAttrs.version;
-    hash = "sha256-tCQ2mPOw7meA7ex7e4BgVco86MNNtxsSC9L6oaVebo4=";
+    hash = "sha256-XVeFpDSgBRbYFKSwDszsKooMC4WJaCxEiCI2yTTCP7I=";
   };
```

Comment:
I guess the version bump is not related to "the fix".  If so, please place it in the second commit to make it clear.

- rime-wanxiang: fix custom patch not taking effect
- rime-wanxiang: 11.3.1 -> 11.3.2